DataFormats/TrackerRecHit2D/src/ProjectedSiStripRecHit2D.cc

```diff
@@ -5,6 +5,6 @@
 
 void ProjectedSiStripRecHit2D::setDet(const GeomDet& idet) {
```

Comment:
Couldn't we just declare `idet` as `const GluedGeomDet&`? It is required to be `GluedGeomDet` by the dynamic cast.

Reply:
It's a virtual function, changing the argument type would lose polymorphism, and also likely force the callers to do a cast.

Reply:
right!